openrouteservice/src/main/java/org/heigit/ors/services/routing/requestprocessors/json/JsonRoutingResponseWriter.java

```diff
@@ -17,6 +17,7 @@
 import com.graphhopper.util.Helper;
 import com.graphhopper.util.shapes.BBox;
 import com.vividsolutions.jts.geom.Coordinate;
+import org.heigit.ors.api.util.SystemMessage;
```

Comment:
Again: core class should not import API classes.

Reply:
ditto.

Reply:
ditto.